src/Illuminate/Database/Query/Builder.php

```diff
@@ -1667,6 +1664,36 @@ public function aggregate($function, $columns = array('*'))
 	}
 
 	/**
+	 * Create a query to compute an aggregate value.
+	 *
+	 * @param  string  $function
+	 * @param  array   $columns
+	 * @return mixed
+	 */
+	protected function aggregateQuery($function, $columns = array('*'))
```

Comment:
The additional method is for clarity. Moreover, we could chose to make it public such that it can be used from the tests and then calling `toSql` to check the query using PHPUnit asserts instead of relying on Mockery to intercept the calls to the database.